charts/grafana/templates/extra-manifests.yaml

```diff
@@ -1,4 +1,8 @@
 {{ range .Values.extraObjects }}
 ---
-{{ tpl (toYaml .) $ }}
+{{- if typeIs "string" . }}
+  {{ tpl . $ }}
```

Comment:
@jkroepke do you know why this needs an extra space before the {{ tpl . $ }} Is this generating a valid template?

Reply:
You are right. We have an issue here. Damit. Could you setup an PR? The space was just cosmetic

Reply:
https://github.com/grafana/helm-charts/pull/3635